receiver/oracledbreceiver/scraper.go

```diff
@@ -269,9 +269,20 @@ func (s *scraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 			now := pcommon.NewTimestampFromTime(time.Now())
 			for _, row := range rows {
 				tablespaceName := row["TABLESPACE_NAME"]
-				err := s.metricsBuilder.RecordOracledbTablespaceSizeLimitDataPoint(now, row["VALUE"], tablespaceName)
-				if err != nil {
-					scrapeErrors = append(scrapeErrors, err)
+				var val int64
+				inputVal := row["VALUE"]
+				valueOk := true
+				if inputVal == "" {
+					val = -1
+				} else {
+					val, err = strconv.ParseInt(inputVal, 10, 64)
+					if err != nil {
+						scrapeErrors = append(scrapeErrors, fmt.Errorf("failed to parse int64 for OracledbTablespaceSizeLimit, value was %s: %w", inputVal, err))
+						valueOk = false
```

Comment:
You could have a `continue` statement here, which would remove the need for `valueOk`.

Reply:
hmm. I guess we can do that yes.

Reply:
ok that's cleaner, thanks!